.travis.yml

```diff
@@ -35,7 +35,11 @@ matrix:
         - 'export PATH=$PATH:~/.bin'
         - nvm install $TRAVIS_NODE_VERSION
         - npm install -g "yarn@${YARN_VERSION}"
+        # trusty comes with pyenv-managed 2.7 and 3.6 - we need python3 -> 3.6 on the PATH for pre-commit to install black during 2.7 tests.
+        - pyenv global 3.6.3 2.7.14
```

Comment:
Why not 3.7?

Reply:
IIRC I shelled into travis and these were the highest versions available. Can recheck when I bump the ubuntu version in a future pr, 3.7 is definitely preferable.